boot/zephyr/main.c

```diff
@@ -198,6 +198,11 @@ void main(void)
     }
 #endif
 
+#ifdef CONFIG_USB
+    rc = usb_enable(NULL);
+    __ASSERT(rc == 0, "Error initializing USB.\n");
+#endif
```

Comment:
Seems that `usb_enable(NULL)` now needs be explicitly called. I would suggest you add this code in:

https://github.com/JuulLabs-OSS/mcuboot/blob/master/boot/zephyr/serial_adapter.c#L196

You probably don't need to test for `CONFIG_USB` there, and it will only be enabled if required.

Reply:
Good recommendation.

Reply:
Yes, USB enable must be called explicitly. This was introduced [here](https://github.com/zephyrproject-rtos/zephyr/pull/20375).  You can find a note about that in [2.2 release notes](https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/releases/release-notes-2.2.rst).
I recommend to call
```
usb_enable(NULL);
```
after 
```
device_get_binding()
```

Reply:
``` C
uart_dev = device_get_binding(CONFIG_USB_CDC_ACM_DEVICE_NAME "_0");
if (uart_dev) {
	rc = usb_enable(NULL);
	if (rc) {
		return -1;
	}
}
```

Reply:
@emob-nordic thanks for your help.

Reply:
Wouldn't this also be required in the `wait_for_usb_dfu()` case as well?  Moving to `serial_adapter.c` wouldn't solve that case ...